backend/python/worker.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+import concurrent.futures
+
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+from activities import FoodDeliveryActivities
+from workflows import OrderWorkflow
+
+TASK_QUEUE = "durable-delivery"
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+
+    activities = FoodDeliveryActivities()
+
+    with concurrent.futures.ThreadPoolExecutor(max_workers=100) as activity_executor:
```

Comment:
Why are you setting the max_workers to 100?  For simplicity, I would create the worker directly, without a the ThreadPoolExecutor, as shown in the docs: https://docs.temporal.io/develop/python/core-application#run-a-dev-worker (or Patrick's saga example: https://github.com/rachfop/temporal-sagas/blob/eb0cbdf3a875b1c57ea3f166d52b61e5037603be/run_worker.py#L44)

Reply:
Since my activities were not asynchronous the worker could not run without an activity_executor. The max workers being 100 was just the solution I found online.